fastlane/Fastfile

```diff
@@ -66,7 +66,13 @@ ALL_LOCALES = [
   { glotpress: "kmr", android: "kmr", promo_config: false},
 ].freeze
 
-RELEASE_NOTES_LOCALES = ALL_LOCALES.reject { |h| h[:google_play].nil? }.map { |h| [h[:glotpress], h[:google_play]] }
+RELEASE_NOTES_LOCALES = ALL_LOCALES
+  .reject { |h| h[:google_play].nil? }
+  .map { |h| [h[:glotpress], h[:google_play]] }
+JP_RELEASE_NOTES_LOCALES = ALL_LOCALES
+  .reject { |h| h[:google_play].nil? }
+  .select { |h| %w[ar de-DE es-ES fr-FR iw-IL id it-IT ja-JP ko-KR nl-NL pt-BR ru-RU sv-SE tr-TR zh-CN zh-TW].include?(h[:google_play]) }
```

Comment:
Would it make sense to add `included_in_jp` or `excluded_from_jp` key to the `ALL_LOCALES` hash? That way we can reuse the information if necessary and it'd all be in one place.